### DISCUSSION TITLE
Instance constantly rescheduling (on x86 and aarch64, but not on darwin)

### DISCUSSION THREAD

**Discussion by truelecter** [cirruslabs/cirrus-ci-docs#1226] [Q&A]
For example, this run:
https://cirrus-ci.com/build/6115111044120576

I was trying different resources per run, but even on 8 CPU it does not have enough time

Do rescheduled build use compute credits, that are introduced for open source projects now? If so, is there anything I can do to fix these reschedules?

**Comment by fkorotkov** [comment]
This should be fixed as of 9am EST today. Sorry for the temporary inconveniences.
